modules/backend/models/accesslog/columns.yaml

```diff
@@ -3,23 +3,34 @@
 # ===================================
 
 columns:
+
+    id:
+        label: backend::lang.access_log.id
+        searchable: yes
+        invisible: true
+        width: 75px
+
     created_at:
         label: backend::lang.access_log.created_at
         searchable: yes
         type: timetense
         width: 160px
 
+    type:
+        label: backend::lang.access_log.type
+        invisible: true
+
+    ip_address:
+        label: backend::lang.access_log.ip_address
+        searchable: yes
```

Comment:
Can you make this one invisible as well?

Reply:
Of course :)

Reply:
I made it.

Reply:
On second review I didn't notice that it was already present and visible; please leave this field visible by default.

Reply:
Now It is visible by default.